### ISSUE TITLE
 SecurityDriven.Core vs SecurityDriven.Inferno

### ISSUE THREAD

**Issue by hippus** [sdrapkin/SecurityDriven.Core#6]
Hi, could you please clarify
- is CryptoRandom in this library different from main lib (SecurityDriven.Inferno) and how?
- which lib is better to use?
Thanks

**Comment by sdrapkin** [comment]
> is CryptoRandom in this library different from main lib (SecurityDriven.Inferno) and how?

Yes, they are different. [SecurityDriven.Inferno] has been designed for .NET Framework 4.x (on Windows) many years ago, and its CryptoRandom implementation is functionally correct but has an older and less performant implementation. [SecurityDriven.Core] is a newer more performant implementation which takes advantage of everything .NET 5+ has to offer.
> which lib is better to use?

If you can use both libraries, prefer [SecurityDriven.Core] for all randomness needs, and use [SecurityDriven.Inferno] for cryptography. If you can only use [SecurityDriven.Inferno] (ex. cannot run .NET 5+), CryptoRandom in Inferno works.
